src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -208,7 +208,7 @@ export class AutoColumnSize extends BasePlugin {
     this.addHook('afterLoadData', () => this.onAfterLoadData());
     this.addHook('beforeChange', changes => this.onBeforeChange(changes));
     this.addHook('afterFormulasValuesUpdate', changes => this.onAfterFormulasValuesUpdate(changes));
-    this.addHook('beforeRender', force => this.onBeforeRender(force));
+    this.addHook('beforeViewRender', force => this.onBeforeRender(force));
```

Comment:
Can we update the methods names or is this a breaking change?
```suggestion
    this.addHook('beforeViewRender', force => this.onBeforeViewRender(force));
```

Since we have `beforeViewRender` and `beforeRender` now, the name `onBeforeRender` may suggest that it's bind to the other one.

Reply:
I agree. Done within e32442e2f3c0bcec211646bcaad7795df29580b2.